example_contracts/typeConversion/unusedArrayConversion.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.8.10;
+
+// SPDX-License-Identifier: MIT
+
+contract WARP {
+  function test() public {
+    uint a;
+    uint8 b;
+    [a,b,3];
```

Comment:
out of interest what is the default type chosen to be and does this work with assignments such as
`uint256[] = [a,b,3]`

Reply:
Just ran an example and saw that it works.